packages/dd-trace/test/appsec/reporter.spec.js

```diff
@@ -360,22 +349,46 @@ describe('reporter', () => {
       Reporter.finishRequest(req, wafContext, {})
 
       expect(web.root).to.have.been.calledOnceWithExactly(req)
-      expect(span.addTags).to.have.been.calledOnceWithExactly({ a: 1, b: 2 })
+      expect(span.addTags).to.have.been.calledWithExactly({ a: 1, b: 2 })
```

Comment:
In the points that we have changed `calledOnceWithExactly` to `calledWithExactly`, could it worth to check first `expect(span.addTags).to.have.been.calledTwice`? 

```suggestion
      expect(span.addTags).to.have.been.calledTwice
      expect(span.addTags).to.have.been.calledWithExactly({ a: 1, b: 2 })
```

Reply:
It was like that in my first commit, but i realized, this test isn't really checking the other call, it's only checking this one, so i'd say it doesn't have to check if it's called twice. I deliberately removed it from my commit because this check is done in a different test

Reply:
to be more clear: this test is unrelated to the second call, so i think if the second call fails, it shouldn't fail this test